charts/seed-controlplane/charts/etcd/templates/etcd.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: druid.gardener.cloud/v1alpha1
+kind: Etcd
+metadata:
+  name: etcd-{{ .Values.role }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    garden.sapcloud.io/role: controlplane
+    role: {{ .Values.role }}
+{{- if .Values.labels }}
+{{ toYaml .Values.labels | indent 4 }}
+{{- end }}
+spec:
+{{- if .Values.podAnnotations }}
+  annotations:
+{{ toYaml .Values.podAnnotations | indent 4 }}
+{{- end }}
+  selector:
+    garden.sapcloud.io/role: controlplane
+    app: etcd-statefulset
+    role: {{ .Values.role }}
+  labels:
+    garden.sapcloud.io/role: controlplane
+    app: etcd-statefulset
+    role: {{ .Values.role }}
+    networking.gardener.cloud/to-dns: allowed
+    networking.gardener.cloud/to-public-networks: allowed
+    networking.gardener.cloud/to-private-networks: allowed
+  etcd:
+    tls:
+      serverTLSSecretRef:
+        name: {{ .Values.tlsServerSecretName }}
+        namespace: {{ .Release.Namespace }}
+      clientTLSSecretRef:
+        name: {{ .Values.tlsClientSecretName }}
+        namespace: {{ .Release.Namespace }}
+    metrics: {{ .Values.metrics }}
+    version: {{ .Values.etcdVersion }}
+    defragmentationSchedule: '0 */24 * * *'
```

Comment:
Current defrag schedule/period is thrice a day and no change recommended there. So, please change it to '0 0 */3 * *'

Reply:
I have updated to have defragment schedule and snapshot schedule from maintenance window if not provided already.